resources/views/settings/account.blade.php

```diff
@@ -149,4 +207,29 @@
             </div>
         </div>
     </div>
+
+    <div class="modal fade" id="deactivate-2fa" tabindex="-1" role="dialog" aria-labelledby="deactivate2FALabel">
```

Comment:
Only render this modal if the user has 2FA enabled, to reduce the page size for most users.